docker-compose.yml

```diff
@@ -11,4 +11,4 @@ services:
     environment:
       - DEBUG_MODE=True
       - SECRET_KEY=xxxxx
-      - ALLOWED_HOSTS=localhost,127.0.0.1
+      - ALLOWED_HOSTS=localhost,127.0.0.1,0.0.0.0
```

Comment:
Leaving this line to @mhall119

Reply:
See previous explanation of what should be in ALLOWED_HOSTS